src/index.html

```diff
@@ -11,7 +11,144 @@
+        <label for="fname">Name:</label>
+        <input
+          type="text"
+          id="fname"
+          name="fname"
+          required
+          autocomplete="off"
+        ><br>
+        <label for="age">How old are You?</label>
+        <input
+          type="number"
+          id="age"
+          name="age"
+          value="12"
+          min="1"
+          max="100"
+          multiple size="2"
+        ><br>
+        <label for="birthd"> Full date of birth:</label>
+        <input
+          type="date"
+          id="birthd"
+          name="birthd"
+          placeholder="mm/dd/yyyy"
+
+        ><br>
+        <label for="temps">I accept the term of the agreement</label>
+        <input
+          type="checkbox"
+          id="temps"
+          name="temps"
+          required
+        >
+      </fieldset>
+      <fieldset class="reg_info">
+        <legend>Registration</legend>
+        <label for="email">E-mail:</label>
+        <input
+          type="email"
+          id="email"
+          name="email"
+          placeholder="email@example.com"
+          required
+        ><br>
+        <label for="password">Password:</label>
+        <input
+          type="password"
+          id="password"
+          name="password"
+          minlength="4"
+          maxlength="16"
+          required
+        >
+      </fieldset>
+      <fieldset class="about_you">
+        <legend>An interesting fact about you!</legend>
+        <label for="love">Do you love cats?</label>
+        <input
+          type="radio"
+          id="yes"
+          name="love"
+          value="yes"
+        >
+        <label for="yes">Yes</label>
+        <input
+          type="radio"
+          id="no"
+          name="love"
+          value="no"
+        >
+        <label for="no">No</label><br>
+        <label for="color">What is your favorite color?</label>
+        <input
+          type="color"
+          id="color"
+          name="color"
+        ><br>
+        <label for="sleep">What time do you go to bed?</label>
+        <input
+          type="time"
+          id="sleep"
+          step="2"
+        ><br>
+        <label for="car">What are your favorite brand of cars?</label>
+        <select
+          name="car"
+          id="car"
+          multiple size="4"
+        >
+          <option value="BMW">BMW</option>
+          <option value="Mersedes">Audi</option>
+          <option value="Tesla">Lada</option>
+        </select><br>
+        <label for="rate">How do you rate our work?</label>
+        <input
+          type="range"
+          id="rate"
+          name="rate"
+          minlength="0"
+          maxlength="10"
+          value="0"
+        >
+      </fieldset>
+      <fieldset>
+        <legend>Additional info:</legend>
+        <label for="comments">Comments:</label>
+        <textarea
+          name="comments"
+          id="comments"
+          cols="20"
+          rows="2"
+        >
+        </textarea><br>
```

Comment:
```suggestion
        <textarea
          name="comments"
          id="comments"
          cols="20"
          rows="2"
      ></textarea><br>
```

it should fix extra spaces in textarea field

Reply:
done